### PR TITLE
[feat][ABE-32] Added new tables and columns

### PR DESCRIPTION
**Pull request by enavarroAK**
### Changelog
- Created tables for directions (province, cantons, districts, and directions)
- Created management category table
- Updated user table adding new columns
- Updated shelter table adding new columns 

### Tickets
https://github.com/akurey/animal-rescue-backend/issues/29
https://github.com/akurey/animal-rescue-backend/issues/32
https://github.com/akurey/animal-rescue-backend/issues/33

### CODE REVIEW

**Review by jsalazarAK — APPROVED** [review]

File: database/004 - add_new_tables_and_columns.sql [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  "name" character varying(50) NOT NULL,
+  "canton_id" int NOT NULL,
+  "created_at" timestamp DEFAULT NOW() NOT NULL,
+  "updated_at" timestamp DEFAULT NOW() NOT NULL,
+  "is_deleted" bit DEFAULT '0' NOT NULL
+);
+
+CREATE TABLE "AP_Directions" (
+  "id" BIGSERIAL PRIMARY KEY NOT NULL,
+  "exact_direction" character varying(50) NOT NULL,
```
Comment (rnavarroSF) [inline database/004 - add_new_tables_and_columns.sql:29]:
```suggestion
  "exact_direction" text NOT NULL,
```

Especially with Costa Rican directions 😂

File: database/004 - add_new_tables_and_columns.sql [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  ADD "identification" character varying(50) NOT NULL,
+  ADD "collegiate_number" character varying(50) NULL,
+  ADD "sinac_registry" character varying(50) NOT NULL;
+
+ALTER TABLE "AP_Shelters"
+  ADD "trade_name" character varying(50) NULL,
+  ADD "management_category_id" int NULL,
+  ADD "sinac_resolution_number" character varying(50) NULL,
+  ADD "direction_id" int NULL,
+  ADD "phone" character varying(50) NULL,
```
Comment (rnavarroSF) [inline database/004 - add_new_tables_and_columns.sql:60]:
```suggestion
  ADD "trade_name" character varying(100) NULL,
  ADD "management_category_id" int NULL,
  ADD "sinac_resolution_number" character varying(20) NULL,
  ADD "direction_id" int NULL,
  ADD "phone" character varying(20) NULL,
```

File: database/004 - add_new_tables_and_columns.sql [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+  "name" character varying(50) NOT NULL,
+  "created_at" timestamp DEFAULT NOW() NOT NULL,
+  "updated_at" timestamp DEFAULT NOW() NOT NULL,
+  "is_deleted" bit DEFAULT '0' NOT NULL
+);
+
+ALTER TABLE "AP_Users"
+  ADD "identification" character varying(50) NOT NULL,
+  ADD "collegiate_number" character varying(50) NULL,
+  ADD "sinac_registry" character varying(50) NOT NULL;
```
Comment (rnavarroSF) [inline database/004 - add_new_tables_and_columns.sql:53]:
```suggestion
  ADD "identification" character varying(20) NOT NULL,
  ADD "collegiate_number" character varying(20) NULL,
  ADD "sinac_registry" character varying(20) NOT NULL;
```

**Review by rnavarroSF — CHANGES_REQUESTED** [review]
Minor changes

**Review by rnavarroSF — APPROVED** [review]
🚀

**Pull request merged**